api/src/main/java/org/openmrs/module/m2sysbiometrics/util/M2SysSearchUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.openmrs.module.m2sysbiometrics.util;
+
+import java.util.List;
+import org.openmrs.module.m2sysbiometrics.bioplugin.BioServerClient;
+import org.openmrs.module.m2sysbiometrics.model.M2SysCaptureResponse;
+import org.openmrs.module.m2sysbiometrics.model.M2SysResults;
+import org.openmrs.module.m2sysbiometrics.xml.XmlResultUtil;
+import org.openmrs.module.registrationcore.api.biometrics.model.BiometricMatch;
+import org.openmrs.module.registrationcore.api.biometrics.model.BiometricSubject;
+
+public final class M2SysSearchUtil {
```

Comment:
I wonder if wouldn't it be better to make this class as a service -> currently we have a RegistrationService so SearchService would keep the consistency of architecture. However, there are no beans in this util class needed so far -> spring component is not necessary here

Reply:
It makes sense. I will change it.